content/documentation/pages/5-feature-guides/2-batch/2-scheduling.md

```diff
@@ -1,7 +1,69 @@
 ---
 path: 'feature-guides/batch/scheduling/'
 title: 'Scheduling Batch Jobs'
-description: 'Scheduling Batch jobs'
+description: 'Scheduling Batch Jobs'
 ---
 
-TODO: Schedule outline
+# Batch Job Scheduling
+
+In [Batch Developers Guide](%currentPath%/batch-developer-guides/) we showed you how to build batch applications and launch them in an adh-hoc fashion.  
+But typically batch are launched at as scheduled time or based on some event.
+In this section we will show you you how you can schedule the launch of your task events using Spring Cloud Data Flow.
+
+## Spring Cloud Data Flow Scheduling Overview
+
+Spring Cloud Data Flow lets a user schedule the launching of tasks via a cron expression. A schedule can be created through the RESTful API or the Spring Cloud Data Flow UI.
+Spring Cloud Data Flow will schedule the execution of its tasks via a scheduling agent that is available on the cloud platform.
+![Scheduling Architecture](images/SCDF-scheduling-architecture.png)
+When using the Cloud Foundry platform Spring Cloud Data Flow will use the PCF Scheduler.
+When using Kubernetes, CronJobs will be used.
+
+## Scheduling a Batch Job
+
+Make sure you have registered your timestamp application and created a definition for it as discussed in the [Getting Started Guide](%currentPath%/batch-developer-guides/getting-started/).
+So using the UI we will schedule the `timestamp-task` by pressing the `drop down` button and selecting the "Schedule Task" option as highlighted below:
+![Create Schedule](images/SCDF-schedule-timestamp.png)
+
+So for this scenario we want to launch this application once a minute. To do this fill out the Schedule setup page as follows:
+
+<!--TABS-->
+
+<!--CloudFoundry-->
+
+In the sample below we set the name of the schedule to `timestamp-task-once-a-minute` and the cron expression to `*/1 * ? * *`.  
+Notice that the cron expression is expressed in the format used by Quartz.
```

Comment:
Should we point to any CF docs that get into cron and the acceptable formats?

Reply:
There is no documentation on the formatting of cron expressions for PCF Scheduler.   They only have a few examples.

Reply:
But I'll refer to what they have.

Reply:
done